contributors/devel/running-locally.md

```diff
@@ -121,10 +121,10 @@ Note the difference between a [container](https://kubernetes.io/docs/user-guide/
 and a [pod](https://kubernetes.io/docs/user-guide/pods/). Since you only asked for the former, Kubernetes will create a wrapper pod for you.
 However you cannot view the nginx start page on localhost. To verify that nginx is running you need to run `curl` within the docker container (try `docker exec`).
 
-You can control the specifications of a pod via a user defined manifest, and reach nginx through your browser on the port specified therein:
+You can control the specifications of a pod via a user defined manifest, and reach nginx through your browser on the port specified there in:
```

Comment:
I think therein is one word: https://www.merriam-webster.com/dictionary/therein :)

Reply:
Also, this line won't hold true if we change it to `multi-pod.yaml`.